Add EmberEurope Q1 2024 (#2338)
mainmatter/mainmatter.com: src/calendar/2024-03-21-embereurope-remote.md

---

> title: "Ember Europe Q4 Meetup"

Reviewer: ```suggestion
title: "Ember Europe Q1 Meetup"
```

image: "/assets/images/calendar/2024-03-21-embereurope-remote/embereurope.jpg"
location: Remote
url: https://www.meetup.com/ember-europe/
kind: meetup
color: purple
---

> The first Ember Europe meetup of 2024 is taking place remotely on March21st. Our

Reviewer: ```suggestion
The first Ember Europe meetup of 2024 is taking place remotely on March 21st. Our
```

Chris Manson will give an update on the progress made with the Embroider

> initiative. The Ember Europe meetup series brings together the Ember community

Reviewer: ```suggestion
initiative and show the current state of Vite support with Embroider. The Ember Europe meetup series brings together the Ember community
```
